ts/sagas/backendInfo.ts

```diff
@@ -22,11 +22,16 @@ const BACKEND_INFO_RETRY_INTERVAL = 60 * 60 * 10 * 1000;
 function* backendInfoWatcher(): IterableIterator<Effect> {
   const backendPublicClient = BackendPublicClient(apiUrlPrefix);
 
-  function getServerInfo(): Promise<BasicResponseType<ServerInfo> | undefined> {
+  function getServerInfo(): Promise<
+    Either<Error, BasicResponseType<ServerInfo>>
```

Comment:
this should probably be `t.Validation<BasicResponseType<ServerInfo> | undefined>` where `t` is `import * as t from "io-ts";`

see [here](https://github.com/teamdigitale/io-ts-commons/blob/master/src/requests.ts#L105)